TTKDTrainingApp/App.js

```diff
@@ -1,17 +1,55 @@
-import React from 'react';
-import {SafeAreaView, StyleSheet, ScrollView, StatusBar} from 'react-native';
+import React, {useEffect} from 'react';
+import 'react-native-gesture-handler';
+import {createAppContainer} from 'react-navigation';
+import {createStackNavigator} from 'react-navigation-stack';
+import {Provider, useDispatch} from 'react-redux';
 
+import store from './src/redux/store';
 import HomeScreen from './src/components/HomeScreen';
+import ReactCamera from './src/components/ReactCamera';
+import VideoWithControls from './src/components/VideoWithControls';
+import Content from './src/components/Content';
+import {setHomeScreenData} from './src/redux/actions';
 
-const App: () => React$Node = () => {
-  return (
-    <React.Fragment>
-      <StatusBar barStyle="dark-content" />
-      <SafeAreaView>
-        <HomeScreen />
-      </SafeAreaView>
-    </React.Fragment>
-  );
+const AppNavigator = createStackNavigator(
+  {
+    HomeScreen: HomeScreen,
+    ReactCamera: ReactCamera,
+    VideoWithControls: VideoWithControls,
+    Content: Content,
+  },
+  {
+    initialRouteName: 'HomeScreen',
+    headerLayoutPreset: 'center',
+    defaultNavigationOptions: {
+      headerStyle: {
+        backgroundColor: 'gray',
```

Comment:
nippick/question: is styling supposed to go in a css file or naw ?

Reply:
In the docs FaceBook doesn't specify that it should be in a separate file, and many examples use styles in the same file so I went that way. It has the added benefit of keeping the file structure slightly cleaner without having Components and their Style files inside folders